README.md

```diff
@@ -93,7 +93,7 @@ Make sure you have docker with docker-compose installed, then run the following
 - copy `dotenv` file into `.env` and set variables as needed (the SHA-256 jwt token is needed)
 - run `make`
 - run `docker compose up -d database` (wait for some seconds the very first time to build the DB)
-- (optional - demo data) run `docker compose run --rm oh-database-init`
+- (optional - demo data after set the database container) run `docker compose run --rm oh-database-init`
```

Comment:
It appears that you are still mixing two different changes into this PR. These 3 files above should not be included.